devnet/README.md

```diff
@@ -0,0 +1,22 @@
+# devnet
+
+Runs a Lotus daemon and miner to be used for boost integration tests.
+
+To run it locally, install ./cmd/devnet and the lotus binaries into your $PATH, and
+simply run one of the integration tests - it should start devnet automatically.
+
+### Building and installing Lotus-related binaries
+
+	git clone https://github.com/filecoin-project/lotus
+	cd lotus
+	git checkout master # or whichever version
+	make debug
+	sudo make install
+	install -C ./lotus-seed /usr/local/bin/lotus-seed
+
+### Clean up state directories
+
+Make sure your $LOTUS_PATH and $LOTUS_MINER_PATH are clean before running devnet:
+
+	rm -rf ~/.lotus
+	rm -rf ~/.lotusminer
```

Comment:
I wonder if we should use non-default directories for this so people don't accidentally over-write their local state for their production miner